pmm-app/tests/verifyPMMDemoPermissionChecks_test.js

```diff
@@ -39,11 +39,20 @@ Scenario(
 );
 
 Scenario(
-  'PMM-T367 - Verify PMM Database checks page doesnt display checks @not-pr-pipeline @pmm-demo',
+  'PMM-T367 - Verify PMM Database checks page doesnt display checks @not-pr-pipeline [critical] @pmm-demo @not-ui-pipeline',
   async (I, pmmDemoPage, databaseChecksPage) => {
     const text = 'Insufficient access rights.';
     I.amOnPage(pmmDemoPage.url + databaseChecksPage.url);
     I.waitForVisible(pmmDemoPage.fields.dbCheckPanelNoAccess, 30);
     await pmmDemoPage.checkDBPanelText(text);
   }
 );
+
+// Need to skip for update to be available on pmmdemo
```

Comment:
pmmdemo does not have up-to-date code?

Reply:
I think those data-qa attributes were added to 2.9.1 and demo has 2.9.0 and verify Post update always takes the latest locators.